tests/chart_tests/test_astronomer_namespace_pools.py

```diff
@@ -274,6 +274,10 @@ def test_astronomer_namespace_pools_create_rbac_mode_is_disabled(self, kube_vers
             show_only=[
                 "charts/astronomer/templates/commander/commander-role.yaml",
                 "charts/astronomer/templates/commander/commander-rolebinding.yaml",
+                "charts/kube-state/templates/kube-state-rolebinding.yaml",
+                "charts/kube-state/templates/kube-state-role.yaml",
+                "charts/astronomer/templates/config-syncer/config-syncer-role.yaml",
+                "charts/astronomer/templates/config-syncer/config-syncer-rolebinding.yaml",
```

Comment:
this tests make sure of validating the namespace pools test cases

Reply:
rest of the cases are already covered by our older test cases

Reply:
yeah, looks good